src/main/java/strman/Strman.java

```diff
@@ -1334,5 +1336,40 @@ private static long countSubstr(String value, String subStr, boolean allowOverla
         return countSubstr(value.substring(offset), subStr, allowOverlapping, ++count);
     }
 
+    /**
+     * Aggregates the contents of n strings into a single list of tuples.
+     *
+     * @param inputs A list of strings.
+     * @return A list of strings if there are enough elements in the input and none of them is null or empty.
+     * An empty list otherwise.
+     */
+    public static List<String> zip(String... inputs) {
+        if (inputs.length < 2) {
+            return emptyList();
```

Comment:
@Cs4r ok let me explain because I think what I said make sense. Scala's list function is called on a list, so you have two lists two work with. One with length 0 and second with length 3. The way zip works is that it picks minimum of the two so you get empty list. 

The code that you have written allows user to pass single String element so I expect it should just give me single element list. Python's zip implementation is similar to what you have written and this is what it does

```python
zip([1,2,3])
[(1,), (2,), (3,)]
```